CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 1.18.2
+
+- Upgrade IotAppKit dependency from [10.11.0 to 10.13.1](https://github.com/awslabs/iot-app-kit/compare/ts-config-v10.11.0...ts-config-v10.13.1)
+- Adding basis universal transcoder to support DRACO compressed tiles
+
```

Comment:
These other changes can be added to the changelog as well.
https://github.com/grafana/grafana-iot-twinmaker-app/pull/306
https://github.com/grafana/grafana-iot-twinmaker-app/pull/307

Reply:
updated Changelog to include these as well 👍